filters/include/pcl/filters/passthrough.h

```diff
@@ -76,7 +77,7 @@ namespace pcl
     * \author Radu Bogdan Rusu
     * \ingroup filters
     */
-  template <typename PointT>
+  template <typename PointT, typename Executor = inline_executor<oneway_t, single_t, blocking_t::always_t>>
```

Comment:
I'm against this style. I'd prefer if only the `filter` function had the template, not the entire class.